examples/persistent-caching/webpack.config.js

```diff
@@ -1,9 +1,11 @@
 const path = require("path");
 module.exports = (env = "development") => ({
 	mode: env,
+	infrastructureLogging: {
+		level: "verbose"
+	},
 	cache: {
 		type: "filesystem",
-		cacheDirectory: path.resolve(__dirname, ".cache"),
-		loglevel: "warning"
+		cacheDirectory: path.resolve(__dirname, ".cache")
```

Comment:
Better use `find-cache-dir` here, to avoid difference places for cache

Reply:
The default uses find-cache-dir. This is only an example